src/ostree/ot-remote-builtin-delete-cookie.c

```diff
@@ -41,7 +41,7 @@ gboolean
 ot_remote_builtin_delete_cookie (int argc, char **argv, GCancellable *cancellable, GError **error)
 {
   g_autoptr(OstreeRepo) repo = NULL;
-  g_autoptr(GOptionContext) context = g_option_context_new ("NAME DOMAIN PATH COOKIE_NAME- Remote one cookie from remote");
+  g_autoptr(GOptionContext) context = g_option_context_new ("NAME DOMAIN PATH COOKIE_NAME- Remove one cookie from remote");
```

Comment:
It looks good overall! The only issue I have is that the description string is now duplicated twice right? Once in the struct and once in the parameter string?

Maybe let's pass the struct to the command so that `ostree_option_context_parse` can set it as the summary? Similar to what we do in rpm-ostree.

Reply:
yea, we could do like https://github.com/projectatomic/rpm-ostree/commit/12c34bb2491a07079c911ef26401fee939e5573c.

But it kinda becomes tricky when we have multiple type of commands included in Ostree. One for `OstreeAdmin`, `OstreeAdminInstutil`, `OstreeRemote`, and things become a bit messier when we have two different parsing method. `ostree_option_context_parse` and `ostree_admin_option_context_parse`. So, going through a similar `invocation` line as rpm-ostree may involve many changes/refactoring to the existing code.

But, it may be not that difficult if we can merge/change all type of different commands in ostree into one whole `OstreeCommand`, and then passing the struct along won't be that hard.  But that also involves changing a lot of code :(, since I am a newbie here, I am inexperienced about whether or not it is worth to do so :p. But I think for now will just go ahead and have it a try :)